docs/docs/customization/entity_fields.md

```diff
@@ -0,0 +1,37 @@
+If you have whitelisted entity fields in Lytics, you can include them in your message, headline, or image field using a simple template that pathfora will recognize and replace with field data. These dynamic field templates allow you to further personalize the customer experience of your module.
+
+# Entity Field Templates
+
+Any field that is available via `window.lio.data` can be placed into a module's message, headline, or image. For example, if you wanted to include the user's name in the headline of a module:
+
+```javascript
+var module = new pathfora.Message({
+  id: 'welcome-back',
+  layout: 'modal',
+  headline: 'Welcome back, {{name}}.',
+  msg: 'Thanks for returning. Please check out our new products.'
+});
+
+pathfora.initializeWidgets([module]);
+```
+
+Pathfora will attempt to replace `{{name}}` with the value of `window.lio.data.name`. Similarly, if you wanted to include a subfield, you can use a dot notation in the template. `{{last_cta.resource}}` would be replaced with the value of `window.lio.data.last_cta.resource`.
+
+If the data isn't present for the user viewing the modal there are a couple options available to us to control the behavior of the module. By default if there is no fallback defined for the field, pathfora will not display any module including data does not exist for the user. This behavior can be changed with the [showOnMissingFields](/display_conditions.md#showonmissingfields) display condition.
+
+# Defining a Fallback (default) Value
+
+Alternatively, we can define a fallback or "default" value to be shown in case the user is missing the data. To define a fallback value, place it after the field name and a `|` character in the template. Or, continuing the example from above:
```

Comment:
` template. Or, continuing the example from above:` => ` template. Building from the previous example:`